src/scripts/pypa-build.sh

```diff
@@ -0,0 +1,14 @@
+main() {
+  local -a build_args
+  build_args=(
+    --outdir "$PARAM_OUTDIR"
+  )
+  [[ $PARAM_SDIST ]] && build_args+=( --sdist )
+  [[ $PARAM_WHEEL ]] && build_args+=( --wheel )
+  [[ $PARAM_SKIP_DEPENDENCY_CHECK ]] && build_args+=( --skip-dependency-check )
+  [[ $PARAM_NO_ISOLATION ]] && build_args+=( --no-isolation )
+  python -m build "${build_args[@]}" .
```

Comment:
While it doesn't hurt to do this, I would rather wrap just the command being executed. It will make it for a cleaner console, and you will still have the info of what args were used.